src/components/TextInput/index.tsx

```diff
@@ -1,70 +1,81 @@
-/** @jsx jsx */
 import * as React from "react";
-import { jsx } from "@emotion/core";
-import Text from "../Text";
+import styled from "lib/styled";
```

Comment:
I'm not crazy about generic "Refactor" commits. I'd much rather prefer atomic commits which eventually achieve the equivalent of this commit. They're easier to review as well as highlight the reason why the refactor was necessary/useful.

I don't want to hold up this PR because of that. Just food for thought. We can chat more in depth about my comment if you'd like.